### ISSUE TITLE
Doesn't wanna work

### ISSUE THREAD

**Issue by DanThyMan** [forestrf/Custom-Blender-Unity-Import-Settings#1]
Copied and pasted the link and it says

'[Package Manager Window] Error adding package: [Package Manager Window] Error adding package: https://github.com/forestrf/Custom-Blender-Unity-Import-Settings.git. UnityEditor.EditorApplication:Internal_CallUpdateFunctions () .
UnityEditor.EditorApplication:Internal_CallUpdateFunctions ()'

**Comment by forestrf** [comment]
It does work for me, so I went to https://docs.unity3d.com/Manual/upm-ui-giturl.html to see what could the problem be.
Sadly, the error Unity reported to you isn't listed there, but maybe it is because you don't have Git installed (I don't know so I'm assuming that may be the case). If so, can you download and install it from https://git-scm.com/downloads and then try again?

**Comment by DanThyMan** [comment]
imma be honest i know have no idea what Git does. I downloaded now it and now i have no clue what it does lol. My first time using Blender and Unity out and this has me lost.

**Comment by forestrf** [comment]
Git is used for several things related to working with others and keeping track of what changes were made. For Unity to download a package stored here (GitHub) and other places like this one that use Git, you need git installed on your computer.
Given its your first time using Blender, and specially if you are using the latest version, this asset is overkill and you shouldn't even need it, this is used for fixing a problem with an old Blender version and to customize default automatic export settings from blender files that you have stored inside the Assets folder of Unity (instead of exporting the 3d model to .fbx and storing that in Assets, aka, letting Unity import the .blend file directly).
TL;DR, you very probably don't need this at all, just drag and drop your blender file inside Unity and it should just work.

**Comment by DanThyMan** [comment]
I tried that before and it didn't work, but somehow magically after reading this it does lol. Thanks for responding anyway preciate' it. Though it does seem to simplify the textures idk how that'll work later
